pkg/daemon/controller/controller_test.go

```diff
@@ -129,6 +131,93 @@ func TestResolveNodeVariables(t *testing.T) {
 	}
 }
 
+func TestResolvePowershellVariables(t *testing.T) {
```

Comment:
The introduction of this unit test function was the only merge conflict. Everything else was cleanly cherry-picked.

Reply:
Why is the unit test being introduced in this PR? It has nothing to do with it

Reply:
Seems the correct thing to do here is drop this function completely

Reply:
Keeping it prevents future merge conflicts related to this func. But I see your point -- the PR which first introduced function was not backported so why include it at all. I'll adjust the commit